core-web/libs/sdk/client/src/lib/client/content/methods/get-collection/get-collection.ts

```diff
@@ -0,0 +1,285 @@
+import { Equals } from '../../../../query-builder/lucene-syntax';
+import { QueryBuilder } from '../../../../query-builder/sdk-query-builder';
+import { CONTENT_API_URL } from '../../shared/const';
+import { GetCollectionResponse, QueryBuilderCallback, SortByArray } from '../../shared/types';
+import { sanitizeQueryForContentType } from '../../shared/utils';
+
+/**
+ * 'GetCollection' Is a Typescript class that provides the ability build a query to get a collection of content.
+ *
+ * @export
+ * @class GetCollection
+ */
+export class GetCollection {
+    #page = 1;
+    #limit = 10;
+    #depth = 0;
+    #render = false;
+    #sortBy?: SortByArray;
+    #contentType: string;
+    #defaultQuery: Equals;
+    #query?: Equals;
```

Comment:
```suggestion
    #query?: DotQuery | string;
```

`Equals` name is weird for this.

Can we also accept `string` here? that way we will not have another method.

Maybe?